app/scripts/proactive/config.js

```diff
@@ -466,7 +466,7 @@ podman_env_script: `
 import org.ow2.proactive.utils.OperatingSystem;
 import org.ow2.proactive.utils.OperatingSystemFamily;
 
-containerName = "java"
+containerName = "adoptopenjdk/openjdk8:latest"
```

Comment:
you should also replace the singularity container `imageUrl = "docker://java"`

Reply:
ok done